.github/workflows/ci-python.yml

```diff
@@ -119,7 +119,7 @@ jobs:
               if: needs.changes.outputs.python == 'true'
               shell: bash -e {0}
               run: |
-                  mypy --version && mypy -p posthog -p dags -p ee -p products | mypy-baseline filter || (echo "run 'pnpm run mypy-baseline-sync' to update the baseline" && exit 1)
+                  mypy --version && mypy . | mypy-baseline filter || (echo "run 'pnpm run mypy-baseline-sync' to update the baseline" && exit 1)
```

Comment:
Nitpick: Would it add value to have a timeout here? Like, 15 minutes or something? It seems enough for normal analysis, but early enough to notice any issues with performance.

Reply:
I assume, CI also has a 30-min timeout? We would probably never hit that 🤔